### PR TITLE
fix security vulnerability

### LINKED ISSUE #992: upgrade webpack
to fix security vulnerability

### PR DESCRIPTION
**Pull request by zeitlinger**
Closes #992

### CODE REVIEW

File: web-ui/package.json [hunk 1]
```diff
@@ -25,22 +27,15 @@
...
     "printWidth": 80,
     "trailingComma": "es5",
     "singleQuote": true
   },
-  "keywords": [
-    "API",
-    "Linter",
-    "Zally"
-  ],
+  "keywords": ["API", "Linter", "Zally"],
```
Comment (tkrop) [inline web-ui/package.json]: I know this. You revert some changes from your last PR... :+1:

**Comment by tkrop** [comment]
:+1:

**Comment by tkrop** [comment]
I can only imagine how hard this was. The build time for zally in Travis is long.

**Comment by zeitlinger** [comment]
:+1:

File: web-ui/package.json (hunk 1 above)
Comment (zeitlinger) [reply web-ui/package.json]: some autoformatting - haven't paid attention what actually does it.

**Comment by zeitlinger** [comment]
> I can only imagine how hard this was. 

Many thanks to Dmytro Bukhanets - who helped me today!

> The build time for zally in Travis is long.
Yes, adding the caches didn't really help - but this was not my main focus for today.

**Pull request merged**